autonomy/cli/analyse.py

```diff
@@ -115,8 +115,6 @@ def abci_app_specs(
                 click.echo(f"Updated FSM specification for {package}")
                 return
 
-            click.echo("Please provide valid arguments")
-
```

Comment:
this code is logically impossible to reach.
- `all_packages: bool` and `update: bool`
    ```
    if all_packages:
    elif not all_packages and not update:
    elif not all_packages and update:
    else: click.echo("Please provide valid arguments")
    ```

```